x-pack/plugin/ml/src/internalClusterTest/java/org/elasticsearch/xpack/ml/integration/ChunkedTrainedModelRestorerIT.java

```diff
@@ -36,11 +40,11 @@
     public void testRestoreWithMultipleSearches() throws IOException, InterruptedException {
         String modelId = "test-multiple-searches";
         int numDocs = 22;
-        List<String> modelDefs = new ArrayList<>(numDocs);
+        List<BytesReference> modelDefs = new ArrayList<>(numDocs);
 
         for (int i=0; i<numDocs; i++) {
             // actual content of the model definition is not important here
-            modelDefs.add("model_def_" + i);
+            modelDefs.add(new BytesArray(Base64.getEncoder().encode(("model_def_" + i).getBytes(StandardCharsets.UTF_8))));
```

Comment:
It might be nice to have a private method `BytesArray base64Encode(String)` and use it throughout